### DISCUSSION TITLE
How to isolate clients?

### DISCUSSION THREAD

**Discussion by newand89** [complexorganizations/wireguard-manager#317] [Q&A]
Can you tell me how to isolate clients from each other? Thank you in advance.

**Comment by Rebelllious** [comment]
I assume you would need to set subnet mask /32 for the clients in their configs instead of the default /8.
If that does not work, you can attempt it via iptables (granted wg0 is the name of the wireguard interface):
```
iptables -I FORWARD -i wg0 -o wg0 -j REJECT --reject-with icmp-admin-prohibited
ip6tables -I FORWARD -i wg0 -o wg0 -j REJECT --reject-with icmp6-adm-prohibited
```
